databases/week1/homework.sql

```diff
@@ -0,0 +1,75 @@
+-- ques1 : Find out how many tasks are in the task table
+select count(id)
+from task;
+
+
+-- ques2 : Find out how many tasks in the task table do not have a valid due date
+
+select count(title)
+from task
+where due_date is null;
+
+
+-- ques3 : Find all the tasks that are marked as done
+
+select count(title)
+from task
+where status_id = 3 ;
+
+
+-- ques4 : Find all the tasks that are not marked as done
+
+select count(title)
+from task
+where status_id = '1' or status_id = '2';
+
+
+-- ques5 : Get all the tasks, sorted with the most recently created first
+
+select *
+from task
+order by due_date desc;
+
+
+-- ques6 : Get the single most recently created task
+
+select (id)
+,(title)
+,(description)
+,(created)
+,(updated)
+,(due_date)
+,(status_id)
+,(user_id)
```

Comment:
here you don't need parenthesis :)

Reply:
Yes, sorry i don't know why i used it, it works without parenthesis :)

Reply:
all good ;)